cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -76,6 +76,8 @@ message SeismicStore {
     File ingested_file = 6; // The info for the original file, if it was ingested as a file
     VolumeDef inline_volume_def = 7; // Volume definition for the store, indexed on inlines. Maps from an inline to all of its valid crosslines
     VolumeDef crossline_volume_def = 8; // Volume definition for the store, indexed on crosslines. Maps from a crossline to all of its valid inlines
+    TextHeader text_header = 9;
```

Comment:
...why is coverage 12? (or, why is there an "empty space" between tags 8 and 12 here?)

Reply:
(reusing protobuf tags is ... not great.)

Reply:
There doesn't seem to be an obvious reason for it, I'll commit a fix to have coverage as 11

Reply:
I'm sorry I was unclear. The thing I was worried about here was that we would be reusing some tags that had been added at one point and then removed. It shouldn't be a big problem for this protobuf, though, as it has seen very limited use, basically just us and our SDK, so I'm not _very_ worried, but as a rule we should try to keep these things in mind when changing protobuf definitions: https://developers.google.com/protocol-buffers/docs/proto3#updating

Reply:
Alright, I've gone spelunking in git history, and my working hypothesis is that @skairunner just copypasted the `Geometry coverage = 12` line from `message Seismic` above. Nevermind me, carry on. :p